install_config/install/advanced_install.adoc

```diff
@@ -1690,6 +1786,80 @@ ifdef::openshift-origin[]
 endif::[]
 ----
 
+[discrete]
+[[running-the-advanced-installation-tsb]]
+==== Optional: Deploying the Template Service Broker
+
+If you have xref:enabling-service-catalog[enabled the service catalog] and
+want to deploy the xref:configuring-template-service-broker[template service
+broker], run the following manual steps after the cluster installation
+completes:
```

Comment:
seems like something in here should link back to the part of the ansible install that talked about configuring the TSB (setting the namespace it uses) since that's a pre-req step to any of this other stuff.